### DISCUSSION TITLE
获取设置失败： 请稍后，正在加载储存

### DISCUSSION THREAD

**Discussion by attack-flower** [AlistGo/alist#3976] [Q&A] [answered]
有一段时间没打开网页，就会提示这个。刷新一下就好了。
是什么原因吗？我是用的alwaysdata的托管搭的，想问一下有遇到这个问题的吗？
<img src="https://imgur.xiaoao.cyou/images/2023/03/27/iShot_2023-03-27_23.04.50.md.png"/>

**Comment by M-CU** [comment]
我今天遇到跟你一样的问题，不知道该怎么解决

**Comment by woshichenghaibo** [comment]
貌似官方没说这个原因是什么？

**Comment by iowong** [comment]
我一更新就这样 只能卸载重装

**Comment by shdxpd** [comment]
一样的错误，啥都没干就报错

**Comment by xhofe** [comment, marked as answer]
字面意思 加载存储需要时间，等就行了。

另：api.nn.ci被阻断了，所以今天出现的会比较多，默认会重试3次，等加载完把后台的阿里云盘的Oauth令牌链接换成 https://api-cf.nn.ci/alist/ali_open/token 或者 https://api.xhofe.top/alist/ali_open/token 就好了。

**Reply by aioufu** [reply]
刷新令牌怎么获取

**Reply by TomPops** [reply]
> 字面意思 加载存储需要时间，等就行了。
> 
> 另：api.nn.ci 被阻断了，所以今天出现的会比较多，默认会重试3次，等加载完把后台的阿里云盘的Oauth令牌链接换成 https://api-cf.nn.ci/alist/ali_open/token 或者 https://api.xhofe.top/alist/ali_open/token 就好了。

非常感谢，换令牌链接后搞定了！

**Comment by xhofe** [comment]
- #4572

**Comment by o2element** [comment]
其实不用一直等，等了半天也未必能加载出来。直接进后台管理页面：http://alist地址:5244/@manage/settings/site 把阿里相关的存储都删掉，然后刷新一下页面就进去了

**Reply by PieJEed** [reply]
问题是现在连后台管理页面都进不去😥

**Comment by shaohuazi** [comment]
是啊，我一直没搞懂，存储加载不出来就进不去管理页面，进不了管理页面就没法删除加载不出的存储，这个逻辑真的好怪

**Comment by xhofe** [comment]
可以使用`alist storage`子命令查看与禁用相关存储：
```bash
./alist storage
Manage storage

Usage:
  alist storage [command]

Available Commands:
  disable     Disable a storage
  list        List all storages

Flags:
  -H, --height int   Table height (default 10)
  -h, --help         help for storage

Global Flags:
      --data string     data folder (default "data")
      --debug           start with debug mode
      --dev             start with dev mode
      --force-bin-dir   Force to use the directory where the binary file is located as data directory
      --log-std         Force to log to std
      --no-prefix       disable env prefix

Use "alist storage [command] --help" for more information about a command.
```

**Reply by free2w** [reply]
这是正解，进不到浏览器界面时，命令行禁用报错的存储，例如有时候 dropbox，alidrive 会抽风。

作为一个经验，挂 alidrive 的两个盘（两种类型）的时候，会经常出错，超期或者重试次数引发风控等等。
所以禁用它们并且只启用其中一个是安全的做法。

**Comment by ZiyanZhuang** [comment]
5/30 遇到这个问题了，无法登录alist也就无法更改令牌？
ERRO[2025-05-30 00:53:32] failed get enabled storages: Post "https://api.nn.ci/alist/ali_open/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
failed init storage
